PantheonCMD/pcprvalidator.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python3
+
+from pygit2 import Repository
```

Comment:
@adahms 
I can swap it out for subprocess.run and some git command. Considering we don't use python3-pygit2 for anything else, it might be unnecessarily to start using it now. What do you think?